crates/iota-names/src/registry.rs

```diff
@@ -92,16 +92,17 @@ impl TryFrom<MoveObject> for NameRecord {
     }
 }
 
+// TODO I'm thinking of removing this impl
```

Comment:
These 3 functions are not used, I'd just remove them for now

Reply:
Do it